infra/src/main/kotlin/fi/fta/geoviite/infra/publication/PublicationDao.kt

```diff
@@ -166,6 +170,16 @@ class PublicationDao(
 
     fun fetchSwitchPublishCandidates(): List<SwitchPublishCandidate> {
         val sql = """
+            with splits as (
+                select
+                    split.id as split_id,
+                    array_agg(split_relinked_switches.switch_id) as split_relinked_switch_ids
+                from publication.split
+                    inner join publication.split_relinked_switch split_relinked_switches 
+                        on split_relinked_switches.split_id = split.id
+                where split.bulk_transfer_state = 'PENDING' and split.publication_id is null
```

Comment:
Riittäisiköhän tässä oikeastaan pelkkä `split.publication_id is null`-tarkastelu? 🤔 Massasiirtohan suoritetaan vasta itse julkaisun Ratko-puskun jälkeen, niin ainakin nykyisellään ei liene mahdollista että se olisi mitään muuta kuin `PENDING` jos julkaisua ei ole